dashboard/app/models/user_geo.rb

```diff
@@ -22,6 +22,8 @@
 #
 
 class UserGeo < ActiveRecord::Base
+  belongs_to :user
```

Comment:
@aoby Is it best practice to add this matching association even though I'm not using any of the helpers it generates, or should I omit it because it's "unused?"

Reply:
I don't know of guidance either way on that one, and I don't have a strong preference. It shouldn't be any extra overhead to add it since they're lazily loaded.